.changeset/nervous-bees-guess.md

```diff
@@ -0,0 +1,18 @@
+---
+'@primer/react-brand': patch
+---
+
+Added support for emphasized text using `<b>` elements to Bento, RiverBreakout, SectionIntro, Timeline, and Testimonial components.
```

Comment:
Could we mention here that we've done this to ensure the emphasised text is a purely visual treatment and won't be picked up by assistive tech? It might not be immediately clear to reads of the change log entry why we've done this otherwise, and why it's important.

Something like..

```suggestion
Added support for `<b>` elements to `Bento`, `RiverBreakout`, `SectionIntro`, `Timeline`, and `Testimonial` components, as `<em>` is semantically interpreted by screen readers for emphasis, while `<b>` allows duo-tone text to function purely as a visual decoration.
```